pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -1961,6 +2008,40 @@
           %leave  ~
       ==
     --
+  ::  +add-cache: remember a cache mapping
+  ::
+  ++  add-cache
+    |=  [=path =cache-entry]
+    ^-  [(list move) server-state]
+    =/  cache-entries  (fall (~(get of cache.state) path) ~)
+    =.  cache.state  (~(put of cache.state) path [cache-entry cache-entries])
+    `state
+  ::  +del-cache: forget a cache mapping
+  ::
+  ++  del-cache
+    |=  =path
+    ^-  [(list move) server-state]
+    =.  cache.state  (~(put of cache.state) path ~)
```

Comment:
@tacryt-socryp is this what you meant by leaving a `~` in the binding?

Reply:
For now, let's replace the most recent list item with a `~`. (This will require the trie to contain `(list (unit cache-entry))`. This will allow us to have deleted the most recent item and have a different case for *is deleted* vs *does not exist*.